LibLootSummary.lua

```diff
@@ -2,10 +2,26 @@
 -- Distributed under the MIT license (see LICENSE.txt)          --
 ------------------------------------------------------------------
 
-LibLootSummary = { version = "2.2.2" }
-local lls = LibLootSummary
+local ADDON_NAME = "LibLootSummary"
 
---[[ Shortcut to creating a new LibLootSumary.List instance ]]--
+local lls = {}
+LibLootSummary = lls
+
+-- [[ Get AddOn version from manifest file ]]--
+local function GetAddOnVersion()
+	local addOnManager = GetAddOnManager()
+	local numAddOns = addOnManager:GetNumAddOns()
+	local name, author
+	for i = 1, numAddOns do
+		name, _, author = addOnManager:GetAddOnInfo(i)
+		if name == ADDON_NAME then
+			return addOnManager:GetAddOnVersion(i)
+		end
+	end
+end
+lls.version = GetAddOnVersion()
```

Comment:
I'm not a big fan of replacing a hardcoded version with something that eats up CPU cycles every login.  I'd rather one developer spend an extra 10 seconds every 3 months updating the value than thousands of users having to iterate the entire addon list every time they log on.  This isn't even an official part of the library. It's just something I added in for debugging purposes.

Reply:
Fair enough.
I'm just lazy to write something manually because I often forget to change that value.